manual/src/threelang/tfgrid/discourse.md

```diff
@@ -0,0 +1,37 @@
+# Discourse Namespace
+
+- To deploy a discourse instance, use the discourse namespace.
+
+## Create Operation
+
+- action name: !!tfgrid.discourse.create
+- parameters:
+  - model_name [required]
+  - farm_id [optional]
+    - if 0, machines could span multiple nodes on different farms
+  - capacity [required]
+    - a string in ['small', 'medium', 'large'] indicating the capacity of the machines
+    - small: 1 vCPU, 2GB RAM, 10GB SSD
+    - medium: 2 vCPU, 4GB RAM, 50GB SSD
+    - large: 4 vCPU, 8GB RAM, 400GB SSD
```

Comment:
i was thinking about unifying capacity settings to not confuse users. this would make it easier right?
```
- a string in ['small', 'medium', 'large', 'extra-large'] indicating the capacity of the discourse instance
  - small: 1 vCPU, 2GB RAM, 10GB SSD
  - medium: 2 vCPU, 4GB RAM, 50GB SSD
  - large: 4 vCPU, 8GB RAM, 240 SSD
  - extra-large: 8vCPU, 16GB RAM, 480GB SSD
```
maybe we could modify these settings if they are not that good, but i would suggest unifying them for all solutions

Reply:
It would make it easier yes but for some solutions the cpu is not needed as much as for other solutions. So I would still choose depending on the requirements.